ulb.c

```diff
@@ -23,16 +23,32 @@ struct state {
     int nextRS; // new real server index
 };
 
-// packet structure to log load balancing
-struct packet {
-    unsigned char dmac[ETH_ALEN];
-    unsigned char smac[ETH_ALEN];
-    __be32 daddr;
-    __be32 saddr;
-    int associationType; // 0:not set,1:new association, 2:existing association
+// Log structure
+struct logEvent {
+    // code identifing the kind of events
```

Comment:
Shouldn't we add the port number to logEvent?

Reply:
As the idea is to have a minimal logging facilities, that's why I didn't add it for now.
But it is in the improvements list : #21 !

We could discuss about that to know if this should be in the v0.1.0.